### ISSUE TITLE
FYI

### ISSUE THREAD

**Issue by hv15** [BluePython339/Parallel_benchmarker#1]
Hi @BluePython339, I one of Prof. Scholz's PhDs. In case you're interested, I created something similar for my benchmarking as part of my research: https://github.com/HighPerLab/dirtygrapher. I haven't used it in a while as I'm doing other stuff now, but you might find it useful in any case.

**Comment by hv15** [comment]
Just to add, my script does not run anything, instead it parses raw output (for which I used another tool which spits out yaml-formatted log files), and transforms the data before finally plotting it. Ooops, just saw that my documentation is abysmal. The `check/` provides an example of how it works, there is a `config.yaml` that specifies the 'paths' withing the raw data to be parsed, transformed, and plotted.

**Comment by BluePython339** [comment]
Sadly the deadline for my own projects have passed (assuming we passed that is), however do you think it would be interesting to develop my tool a bit more and offer it up for use for next year's parallel course?

**Comment by BluePython339** [comment]
@hv15

**Comment by hv15** [comment]
Yes, I think that would be nice, graphing tends to be tricky (both in how to graph but also in how to display the graph). If choose to develop your tool further, I'd share this with next years students for the course :)

**Comment by BluePython339** [comment]
As soon as the exams are over i'll clean it up a bit and assure it can compare multiple programs (and possibly adding support for MPI)
